releng/gradle-composite/settings.gradle

```diff
@@ -1,12 +1,24 @@
+apply from: 'helper.gradle'
+
 if (!hasProperty('generatePrefs')) {
-    (files { file('../../../').listFiles() })
-    .filter { 
-        File it ->
-        it.isDirectory() && XTEXT_GRADLE_PROJECTS.split(',').any{p->it.name.contains(p)} && new File(it, "settings.gradle").exists()
-    }.collect{ it.path }
-    .toUnique { a, b ->
-        findProjectName(a) <=> findProjectName(b)
-    }.each { it -> includeBuild it }
+    def projects = getProjectsFromLocationProperties()
+    if (!projects.isEmpty()) {
+        projects.each {
+            file ->
+            println("includeBuild ${file}")
```

Comment:
This output should be removed (see [Logging](https://docs.gradle.org/current/userguide/logging.html) for proper logging).